test/bats/cli-test.bats

```diff
@@ -34,6 +34,11 @@ load helpers
     assert_cmd_verify_failure
 }
 
+@test "notation verifier crl test" {
+    run bin/ratify verify -c $RATIFY_DIR/config_notation_crl.json -s $TEST_REGISTRY/notation:crl
+    assert_cmd_verify_success
```

Comment:
wonder if the negative case is already covered.

Reply:
Negative case updated.

Reply:
Negative case added.